README.md

```diff
@@ -56,30 +153,29 @@ See the [LICENSE][license_file] file for the full license.
 [jake]: https://github.com/sonatype-nexus-community/jake
 
 [license_file]: https://github.com/CycloneDX/cyclonedx-python-lib/blob/master/LICENSE
```

Comment:
please _order_ the links back like they were. 
file-links were grouped, shield-links were grouped, external-links were grouped ...